Tests/scripts/gitlab_slack_notifier.py

```diff
@@ -563,19 +556,22 @@ def main():
     shame_message = None
     if options.current_branch == DEFAULT_BRANCH and triggering_workflow == CONTENT_MERGE:
         # We check if the previous build failed and this one passed, or wise versa.
-        list_of_pipelines, commits = get_pipelines_and_commits(gitlab_url=server_url, gitlab_access_token=ci_token,
-                                                               project_id=project_id, look_back_hours=LOOK_BACK_HOURS)
-        pipeline_changed_status, pivot_commit = is_pivot(single_pipeline_id=pipeline_id,
-                                                         list_of_pipelines=list_of_pipelines,
-                                                         commits=commits)
-        logging.info(f'we are investigating pipeline {pipeline_id}')
-        logging.info(f'Pivot commit is {pivot_commit}, pipeline changed status is {pipeline_changed_status}')
-        if pipeline_changed_status is not None:
-            name, email, pr = person_in_charge(pivot_commit)
-            msg = "broke" if pipeline_changed_status else "fixed"
-            shame_message = (f"Hi @{name}, You {msg} the build.", f" That was done in this {slack_link(pr,'PR.')}")
-
-            computed_slack_channel = "test_slack_notifier_when_master_is_broken"
+        list_of_pipelines, list_of_commits = get_pipelines_and_commits(gitlab_client=gitlab_client,
+                                                                       project_id=project_id, look_back_hours=LOOK_BACK_HOURS)
+        current_commit = get_commit_by_sha(commit_sha, list_of_commits)
+        if current_commit:
+            current_commit_index = list_of_commits.index(current_commit)
+            previous_commit = list_of_commits[current_commit_index + 1]
+            current_pipeline = get_pipeline_by_commit(current_commit, list_of_pipelines)
+            previous_pipeline = get_pipeline_by_commit(previous_commit, list_of_pipelines)
+            if current_pipeline and previous_pipeline:
+                pipeline_changed_status = is_pivot(current_pipeline, previous_pipeline)
+                logging.info(
+                    f"""Checking pipeline {pipeline_id},
+                    the commit is {current_commit} and the pipeline change status is: {pipeline_changed_status}""")
```

Comment:
why use `"""` ? and not just `"` 

i don't think you meant it to have some many spaces and a new line in the string.